tsdb/db.go

```diff
@@ -66,20 +77,17 @@ type Options struct {
 	// WALSegmentSize = 0, segment size is default size.
 	// WALSegmentSize > 0, segment size is WALSegmentSize.
 	// WALSegmentSize < 0, wal is disabled.
-	WALSegmentSize int
+	WALSegmentSize units.Base2Bytes
 
 	// Duration of persisted data to keep.
-	RetentionDuration uint64
+	RetentionDuration model.Duration
```

Comment:
This is an opaque number, not a specific duration.

Reply:
Hm. It is used like time retention all the time...

Reply:
If we revert this we would need to name it `RetentionDurationMiliseconds`

Reply:
Oh.. . you want to totally abstract from units. Why is that useful? Isn't it just easier to agree on unit?